src/components/buttons/retry-button.ts

```diff
@@ -1,5 +1,4 @@
-import { RETRY_BUTTON } from "@constants";
-
+import { RETRY_BTN_IMG } from '@constants';
```

Comment:
lets use double quotes just to be consistent for imports

Reply:
Changes applied, thank you.